DllImportGenerator/DllImportGenerator/DllImportGenerator.cs

```diff
@@ -287,7 +288,7 @@ private static bool IsGeneratedDllImportAttribute(AttributeSyntax attrSyntaxMayb
 
                 Debug.Assert(expSyntaxMaybe is not null);
 
-                if (PassThroughToDllImportAttribute(namedArg.Key))
+                if (generateForwarders || PassThroughToDllImportAttribute(namedArg.Key))
```

Comment:
Spell out the fact that "pass through" and "forwarder" are the same concept here.